apps/os/package.json

```diff
@@ -7,8 +7,9 @@
     "cf-typegen": "wrangler types --no-include-runtime",
     "deploy": "doppler run -- pnpm run build && doppler run -- wrangler deploy",
     "dev": "./scripts/dev-wrapper.sh",
-    "evals": "doppler run -- evalite run-once && tsx evals/evalite-exporter.ts",
-    "evals:watch": "doppler run -- evalite watch",
+    "eval": "doppler run -- evalite run-once && tsx evals/evalite-exporter.ts",
```

Comment:
How often does it run each eval? In my experience with evals you always invoke the runner with some argument to indicate how often you want to run the evals?

Or, in more fancy circumstances, you might say something like "Run this eval as many times as is necessary to gain 98% confidence that you have not made eval scores worse than the reference branch, but no more than 50 times"

Reply:
once, but matt himself plans(?) to add it: https://github.com/mattpocock/evalite/issues/12. tbh I see evalite as just a starting point for us, only a small headstart for our own thing. either as a forkable repo, something that is actively developed that we can contribute to, or just a reference implementation for the CRUD needed for a UI + mini DB.